Packs/Coralogix/Integrations/Coralogix.py

```diff
@@ -0,0 +1,447 @@
+            if len(incidents_query.strip()) > 0 and len(incidents_name_field.strip()) > 0:
+                incidents = fetch_incidents(
+                    cgx_private_key=private_key,
+                    cgx_endpoint_url=search_api_url,
+                    incidents_base_query=incidents_query,
+                    application_name=incidents_application_name,
+                    subsystem_name=incidents_subsystem_name,
+                    severity=incidents_severity,
+                    incidents_name_field=incidents_name_field
+                )
+                if demisto.command() == 'clx_fetch_incidents':
+                    demisto.results(incidents)
+                else:
+                    demisto.incidents(incidents)
+            else:
+                if demisto.command() == 'clx_fetch_incidents':
+                    if len(incidents_query.strip()) == 0:
+                        demisto.results("ERROR: Cannot fetch incidents. No incidents query defined.")
+                    if len(incidents_name_field.strip()) == 0:
+                        demisto.results("ERROR: Cannot fetch incidents. The incident_description_field is empty.")
+                else:
+                    LOG("ERROR: Cannot fetch incidents. Either no incidents query defined or the incident_description_field is empty.")
+                    demisto.incidents([])
+        elif demisto.command() == 'clx_tag':
+            if "name" in demisto.args():
+                tag_name = demisto.args()["name"].strip()
+            else:
+                raise Exception("Tag name is missing")
+            if "timestamp" in demisto.args():
+                tag_timestamp = demisto.args()["timestamp"].strip()
+            else:
+                tag_timestamp = ""
+            if "icon_url" in demisto.args():
+                tag_icon_url = demisto.args()["icon_url"].strip()
+            else:
+                tag_icon_url = ""
+            demisto.results(clx_tag_command(
+                private_key=private_key,
+                coralogix_url=tags_api_url,
+                application_name=tag_application_name,
+                subsystem_name=tag_subsystem_name,
+                tag_name=tag_name,
+                tag_timestamp=tag_timestamp,
+                tag_icon_url=tag_icon_url
+            ))
+        elif demisto.command() == 'clx_search':
+            if "query" in demisto.args():
+                query = demisto.args()["query"].strip()
+            else:
+                raise Exception("No query specified")
+            if "exclude" in demisto.args():
+                exclude_columns = demisto.args()["exclude"].strip().split(',')
+            else:
+                exclude_columns = []
+            if "as_table" in demisto.args():
+                results_as_table = demisto.args()["as_table"].strip().lower() in ["true"]
+            else:
+                results_as_table = False
```

Comment:
This is why we have an initial value option for each argument. You can just add it there.